protected/humhub/modules/user/models/User.php

```diff
@@ -276,6 +278,10 @@ public function __get($name)
             return $profile;
         }
 
+        if ($name === 'time_zone' && empty(parent::__get($name))) {
```

Comment:
Maybe a comment would be nice. "e.g. fall back to default".

Reply:
https://github.com/humhub/humhub/pull/7243/commits/8a30aae0e95545692516daf5c5f52cb0851ca2d7#diff-d3a4745ae7c92bb2ed3212a38c146f5f29268cf40a693d007f752f679a1f52abR282